src/main/java/ca/gc/aafc/objectstore/api/respository/ObjectUploadResourceRepository.java

```diff
@@ -0,0 +1,29 @@
+package ca.gc.aafc.objectstore.api.respository;
+
+import java.util.UUID;
+
+import org.springframework.stereotype.Repository;
+
+import ca.gc.aafc.objectstore.api.dto.ObjectUploadDto;
+import ca.gc.aafc.objectstore.api.service.ObjectUploadService;
+import io.crnk.core.queryspec.QuerySpec;
+import io.crnk.core.repository.ReadOnlyResourceRepositoryBase;
+import io.crnk.core.resource.list.ResourceList;
+
+@Repository
+public class ObjectUploadResourceRepository
+    extends ReadOnlyResourceRepositoryBase<ObjectUploadDto, UUID> {
+  private final ObjectUploadService service;
+
+  protected ObjectUploadResourceRepository(ObjectUploadService service) {
+    super(ObjectUploadDto.class);
+    this.service = service;
+  }
+
+  @Override
+  public ResourceList<ObjectUploadDto> findAll(QuerySpec querySpec) {
+    // this.service.findAll(entityClass, where, orderBy, startIndex, maxResult)
+    return null;
```

Comment:
not sure overiding findAll to return null is the right approach.

I don't think you need to override this method at all. if your really want to prevent it you could return a method not allowed exception

Reply:
also we should probably not leave code that is commented out.